auth-api/src/main/java/io/stargate/auth/api/impl/AuthApiServerFactory.java

```diff
@@ -60,7 +63,11 @@ public Server build(Environment environment) {
         Collections.singletonMap(getApplicationContextPath(), applicationHandler);
     final ContextRoutingHandler routingHandler = new ContextRoutingHandler(handlers);
     final Handler gzipHandler = buildGzipHandler(routingHandler);
-    server.setHandler(addStatsHandler(addRequestLog(server, gzipHandler, environment.getName())));
+    if (REQUEST_LOGS_ENABLED) {
+      server.setHandler(addStatsHandler(addRequestLog(server, gzipHandler, environment.getName())));
```

Comment:
So this would disable the logging completely, even for the `500`s, maybe this is fine if we have the full stack trace but it's often quite helpful to have those in the logs imo.

Reply:
I think that this is good enough for the client's feature request.

Reply:
Yes, I guess it's ok to disable the request log completely. We should really have a separate log message with the exception for all 500's, so having a line for the HTTP status is not critical, I think. Besides, 500's can spam the log just as well as 200's in high load tests. This new setting is intended to be used a measure against logs growing too long.